example/lib/main.dart

```diff
@@ -12,11 +9,9 @@ import 'package:flutter_local_notifications/flutter_local_notifications.dart';
 
 const String title = 'FileUpload Sample app';
 final Uri uploadURL = Uri.parse(
-  'https://us-central1-flutteruploadertest.cloudfunctions.net/upload',
+  'http://192.168.0.40:5001/flutteruploader/us-central1/upload',
```

Comment:
I guess this is your localhost? not sure about the tests but if they pass we could change that ip to be something else (localhost?)